### PR TITLE
Updating app and features entries for argotronic.argus monitor

### PR DESCRIPTION
**Pull request by BrandonWanHuanSheng**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

Resolves  #108360 

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/108418)

### CODE REVIEW

**Comment by BrandonWanHuanSheng** [comment]
Hmm, It look like the pipelines is not working.

File: manifests/a/Argotronic/ArgusMonitor/6.2.3.2667/Argotronic.ArgusMonitor.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Automatically updated by the winget bot at 2023/May/29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Argotronic.ArgusMonitor
+PackageVersion: 6.2.3.2667
```
Comment (Trenly) [inline manifests/a/Argotronic/ArgusMonitor/6.2.3.2667/Argotronic.ArgusMonitor.installer.yaml:5]:
```suggestion
PackageVersion: 6.2.03
```

File: manifests/a/Argotronic/ArgusMonitor/6.2.3.2667/Argotronic.ArgusMonitor.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Automatically updated by the winget bot at 2023/May/29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Argotronic.ArgusMonitor
+PackageVersion: 6.2.3.2667
```
Comment (Trenly) [inline manifests/a/Argotronic/ArgusMonitor/6.2.3.2667/Argotronic.ArgusMonitor.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 6.2.03
```

File: manifests/a/Argotronic/ArgusMonitor/6.2.3.2667/Argotronic.ArgusMonitor.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Automatically updated by the winget bot at 2023/May/29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Argotronic.ArgusMonitor
+PackageVersion: 6.2.3.2667
```
Comment (Trenly) [inline manifests/a/Argotronic/ArgusMonitor/6.2.3.2667/Argotronic.ArgusMonitor.yaml:5]:
```suggestion
PackageVersion: 6.2.03
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/a/Argotronic/ArgusMonitor/6.2.03/Argotronic.ArgusMonitor.installer.yaml [hunk 4]
```diff
@@ -8,10 +9,9 @@ InstallerType: nullsoft
 UpgradeBehavior: install
 AppsAndFeaturesEntries:
 - ProductCode: ArgusMonitor
```
Comment (mdanish-kh) [inline manifests/a/Argotronic/ArgusMonitor/6.2.03/Argotronic.ArgusMonitor.installer.yaml:11]:
```suggestion
- ProductCode: ArgusMonitor
  DisplayVersion: 6.2.3.2667
```

Marketing version on the publisher's website is 6.2.03 but version written to Registry (visible in Apps&Features) is 6.2.3.2667

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**